blazeit/aggregation/samplers.py

```diff
@@ -84,7 +84,7 @@ def __init__(self, *args):
         self.var_t = np.var(self.Y_pred)
 
     def reset(self, Y_pred, Y_true):
-        self.cov = np.cov(Y_true[0:100], Y_pred[0:100])[0][1]
+        self.cov = np.cov(Y_true[0:100].astype(np.float32), Y_pred[0:100].astype(np.float32))[0][1]
```

Comment:
The reason I have to do this is because np.cov has a bug (?) where it doesn't handle arithmetic right if the dtype of a numpy array is an object even if it supports all arithmetic operations. https://stackoverflow.com/questions/45670487/numpy-cov-exception-float-object-has-no-attribute-shape

Reply:
Welp numpy, sounds good